pkg/cstor/cstor.go

```diff
@@ -137,7 +145,17 @@ func (p *Plugin) getServerAddress() string {
 
 // getMapiAddr return maya API server's ip address
 func (p *Plugin) getMapiAddr() string {
-	sclist, err := p.K8sClient.Services(operator).List(
+	var openebsNs string
+
+	// check if user has provided openebs namespace
+	if p.namespace != "" {
+		openebsNs = p.namespace
```

Comment:
What will be the value of `p.namespace` if nothing is specified?  This doesn't sound correct.